ArticleService.js

```diff
@@ -0,0 +1,60 @@
+import axios from 'axios';
+
+const URL = 'https://panda-market-api-crud.vercel.app/articles/';
+
+export async function getArticleList({page, pageSize, keyword}) {
+  return fetch(URL,{
+      params: {page, pageSize, keyword}
+    })
+  .then(res => res.json())
+  .then(data => {
+    return data
+  })
+  .catch((e) => {
+    if(!res.ok) {
+    console.log('목록을 가져오는 중 오류가 발생했습니다.');
+  }
+});
+}
+
+export async function getArticle(articleId) {
+  const res = await axios.get(`${URL}${articleId}`)
+  .catch((e) => {
+    console.log('오류가 발생했습니다.');
+    return null;
+  });
+  return res?.data || null;

```

Comment:
거의 모든 응답에 `return res?.data || null;`로 작성해주셨어요.
Article CRUD API의 응답이 에러가 발생했을 경우 null로 세팅되는 것 같아요.
그렇다면 `|| null`은 필요없지 않을까요?

```suggestion
  return res?.data;
```